src/System.Runtime.WindowsRuntime/src/System/Runtime/InteropServices/WindowsRuntime/AsyncInfo.cs

```diff
@@ -44,7 +44,7 @@ public static IAsyncAction Run(Func<CancellationToken, Task> taskProvider)
 
 
         /// <summary>
-        /// Creates and starts an <see cref="Windows.Foundation.IAsyncActionWithProgress{TProgress}"/> instance from a function
+        /// Creates and starts an <see cref="T:Windows.Foundation.IAsyncActionWithProgress{TProgress}"/> instance from a function
```

Comment:
❕ The qualified form uses numeric arity:

```
T:Windows.Foundation.IAsyncActionWithProgress`1
```

📝 Applies to 6 changes in this file